src/EditorFeatures/Core/EditorFeaturesResources.resx

```diff
@@ -936,4 +936,7 @@ Do you want to proceed?</value>
   <data name="external" xml:space="preserve">
     <value>(external)</value>
   </data>
+  <data name="Inline_Parameter_Name_Hints" xml:space="preserve">
+    <value>Inline Parameter Name Hints</value>
```

Comment:
where is this disdplayed.  `Capital Case Is A Bit Weird` versus `Most UI string should be title cased`.

Reply:
It's being displayed on the Tools->Options->Fonts And Colors->Display Items